ci/jenkins/test-scale.sh

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+KIND_CLUSTER=""
+DEFAULT_WORKDIR="/var/lib/jenkins"
+DEFAULT_KUBECONFIG_PATH=$DEFAULT_WORKDIR/kube.conf
+WORKDIR=$DEFAULT_WORKDIR
+KUBECONFIG_PATH=$DEFAULT_KUBECONFIG_PATH
+TESTCASE=""
+TEST_FAILURE=false
+DOCKER_REGISTRY=$(head -n1 "${WORKSPACE}/ci/docker-registry")
+TESTBED_TYPE="legacy"
+IMAGE_PULL_POLICY="Always"
+GOLANG_RELEASE_DIR=${WORKDIR}/golang-releases
+
+CONTROL_PLANE_NODE_ROLE="master|control-plane"
+
+CLEAN_STALE_IMAGES="docker system prune --force --all --filter until=4h"
+CLEAN_STALE_IMAGES_CONTAINERD="crictl rmi --prune"
+PRINT_DOCKER_STATUS="docker system df -v"
+PRINT_CONTAINERD_STATUS="crictl ps --state Exited"
+
+_usage="Usage: $0 [--kubeconfig <KubeconfigSavePath>] [--workdir <HomePath>]
+                  [--testcase <e2e|conformance|networkpolicy>]
+
+Run K8s e2e community tests (Conformance & Network Policy) or Antrea e2e tests on a remote (Jenkins) Windows or Linux cluster.
+
+        --kubeconfig             Path of cluster kubeconfig.
+        --workdir                Home path for Go, vSphere information and antrea_logs during cluster setup. Default is $WORKDIR.
+
+function print_usage {
+    echoerr "$_usage"
+}
+
```

Comment:
There are lots of copy-paste codes in this script, please remove unused codes and also correct the descriptions.

Reply:
This script commit was committed by @XinShuYang , could you @XinShuYang help give more comments about it and I will solve these comments together ASAP.